patches/LEDStrip.cpp

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) Robotec.ai.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#include "LEDStrip.h"
+
+#include <Atom/Feature/CoreLights/PhotometricValue.h>
+#include <AtomLyIntegration/CommonFeatures/CoreLights/AreaLightBus.h>
+#include <AzCore/Component/TickBus.h>
+#include <AzCore/Debug/Trace.h>
+#include <AzCore/Math/Color.h>
+#include <AzCore/Serialization/EditContext.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <ROS2/Communication/QoS.h>
+#include <ROS2/Frame/ROS2FrameComponent.h>
+#include <ROS2/ROS2Bus.h>
+#include <ROS2/Utilities/ROS2Names.h>
+
+namespace Husarion
+{
+    LEDStrip::LEDStrip()
+    {
+        m_topicConfiguration.m_topic = "led_strip";
+        m_topicConfiguration.m_type = "sensor_msgs/msg/image";
+    }
+
+    void LEDStrip::Reflect(AZ::ReflectContext* context)
+    {
+        if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serializeContext->Class<LEDStrip>()
+                ->Version(1)
+                ->Field("Leds", &LEDStrip::m_leds)
+                ->Field("intensity", &LEDStrip::m_intensity)
```

Comment:
```suggestion
                ->Field("Intensity", &LEDStrip::m_intensity)
```
Nit: for consistency